styles.css

```diff
@@ -365,18 +462,17 @@ nav {
 .line {
   margin: 18px 0;
   color: #acb7c3;
-  width: 85%;
+  width: 80%;
 }
 
 .contact-us {
   margin: 0 auto;
   background: #6070ff;
   border-top-left-radius: 50px;
-  background-image: url(/img/contact-form-mobile-bg.png);
+  background-image: url(/img/contact-bg.png);
   background-repeat: no-repeat;
-  background-size: contain; 
+  background-size: contain;
   background-position: right;
-  
 }
```

Comment:
- [ ] Great work making some changes to the `background-image` of the Form section. However, I noticed that the `background-image` used is not the correct one and doesn't correlate with that of the Figma template used. Also, the positioning of the `background-image` is not uniform with that of the Figma template chosen. 👍 

### Your Design

![Screenshot1](https://user-images.githubusercontent.com/77106962/222586926-226759d0-b0b3-41d5-b820-b44deae28b26.png)

### Figma Template

![Screenshot2](https://user-images.githubusercontent.com/77106962/222587275-f0ed5122-f669-4c92-9926-97b4ec54efa1.png)